RecoTracker/TrackProducer/plugins/GsfVertexConstraintProducer.cc

```diff
@@ -18,14 +18,13 @@
 // class decleration
 //
 
-class GsfVertexConstraintProducer: public edm::EDProducer {
+class GsfVertexConstraintProducer: public edm::stream::EDProducer<> {
 public:
   explicit GsfVertexConstraintProducer(const edm::ParameterSet&);
-  ~GsfVertexConstraintProducer();
+  ~GsfVertexConstraintProducer() override;
```

Comment:
```suggestion
  ~GsfVertexConstraintProducer() override = default;
```
and remove the empty definition.